references/recaptcha-enterprise/README.md

```diff
@@ -0,0 +1,186 @@
+# reCAPTCHA Enterprise & Apigee X/hybrid
+
+[Google reCAPTCHA Enterprise](https://cloud.google.com/recaptcha-enterprise)
+can be leveraged by Apigee in order to protect backend systems from fraudulent
+client side activities like **bots**.
+A reCAPTCHA token is silentely and periodically retrieved by a client app and
+transmitted to an Apigee runtime when an API is invoked.
+
+## How it works?
+
+The sharedflow configuration - that is proposed through this Apigee devrel
+reference - is able to get a reCAPTCHA token validation status and a risk score
+from the Google reCAPTCHA Enterprise assessment endpoint.
+The risk score is a decimal value between ```0.0``` and ```1.0```.
+
+The score ```1.0``` indicates that the interaction poses low risk and is very
+likely legitimate, whereas ```0.0``` indicates that the interaction poses high
+risk and might be fraudulent. Between both extremes, the sharedflow's
+processing decides if an API invocation must be rejected or not.
+The minimum score to consider an API call as legitimate is set at ```0.6```:
+this value is configurable.
+
+## Apigee runtime options
+
+The reCAPTCHA enterprise reference can be deployed on both Apigee X and
+hybrid.
+
+## Dependencies
+
+- [Maven](https://maven.apache.org/)
+- [NodeJS](https://nodejs.org/en/) LTS version or above
+- Apigee Evaluation/Trial [Organization](https://login.apigee.com/sign__up)
+- [Google Cloud Platform](https://cloud.google.com/)
+
+This reference leverages Google reCAPTCHA Enterprise. Therefore, a Google
+Cloud Platform (GCP) service account is needed by the Apigee configuration
+to securely invoke the Google reCAPTCHA Enterprise assessment endpoint.
+
+This service account is created during the deployment process on the Google
+Cloud project you are currently using.
```

Comment:
The pipelines.sh will create a SA if it doesn't exist. See below for explanation.